recipe/meta.yaml

```diff
@@ -10,46 +11,145 @@ source:
-    # See https://github.com/ami-iit/ADAM/pull/38#issuecomment-1546909762
-    - pytest ./tests/ -v -k "not gravity"  # [not win]
-    - pytest ./tests/ -v --ignore-glob=*Jax* --ignore-glob=*pytorch*  # [win]
-  requires:
-    - pip
-    - pytest
-    - icub-models
-    - gym-ignition-models
-    - pytorch  # [not win]
-    - jax  # [not win]
-    - idyntree
-    - git
-    - gitpython
+    - python {{ minimum_python_version }}
 
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python {{ minimum_python_version }}
+        - pip
+        - wheel
+        - setuptools
+      run:
+        - python {{ minimum_python_version }}
+        - numpy
+        - scipy
+        - prettytable
+        - urdfdom-py
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax* --ignore-glob=*pytorch* --ignore-glob=*CasADi*
+      requires:
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+
+  - name: {{ name|lower }}-casadi
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - casadi
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax* --ignore-glob=*pytorch*
+      requires:
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - casadi
+
+  - name: {{ name|lower }}-jax
+    build:
+      skip: True  # [win]
+    requirements:
+      host:
+        - python
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - jax
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*pytorch* --ignore-glob=*CasADi*
+      requires:
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - jax
+
+  - name: {{ name|lower }}-pytorch
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - jax2torch
+        - pytorch
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax* --ignore-glob=*CasADi*
+      requires:
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - pytorch
+
+  - name: {{ name|lower }}-all
+    build:
+      skip: True  # [win]
```

Comment:
Could it make sense to also have `adam-robotics-all` also on Windows, and just install all the packages that are available there? Or it may be too confusing?

Reply:
Yes, it makes sense. Thanks!

Reply:
Done in https://github.com/conda-forge/adam-robotics-feedstock/pull/12/commits/611b8fc016d0ca01c49a0c730fbf825b3bc72b26